services/web/.eslintrc

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Why is having this as a separate file and not in package.json better? Is this considered to be the trend / better practice? Trying to understand

Reply:
I feel like it's common practice nowadays. Not a big deal but just a bit of separation of concerns to keep `package.json` from becoming massive.

Reply:
i think you are right, better to split out, i do kind of hate all these files but yeah its more common to see that pattern.

Reply:
Makes sense. Proceed!